inst/input/hector-gcam.ini

```diff
@@ -139,6 +139,7 @@ enabled=1
 S=3.0 				; equilibrium climate sensitivity for 2xCO2, degC
 diff=2.3			; ocean heat diffusivity, cm2/s
 alpha=1.0     ; scaling factor for aerosol forcing
+volscl=1.0    ; scaling factor for volcanic forcing
```

Comment:
Perhaps both scaling factors' comments should note centered around 1, i.e. 1.0 = no scaling?